src/index.js

```diff
@@ -0,0 +1,81 @@
+import './style.css';
+
+const iconElement = document.querySelector('.weather-icon');
+const tempElement = document.querySelector('.temperature-value p');
+const descElement = document.querySelector('.temperature-description p');
+const locationElement = document.querySelector('.location p');
+const notificationElement = document.querySelector('.notification');
+
+const weather = {};
+
+weather.temperature = {
+  unit: 'celsius',
+};
+
+const KELVIN = 273;
+const key = '99f15a158a869cfdd635370f80c21b6b';
+
+function displayWeather() {
```

Comment:
Let's use an arrow function instead.

> It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

Let's do this for all other occurrences. :relaxed: